app/pages/cif/projects.tsx

```diff
@@ -81,14 +81,14 @@ export function Projects({ preloadedQuery }: RelayProps<{}, projectsQuery>) {
       <h1>Projects</h1>
       {query.allProjects.edges.length === 0 && <p>None</p>}
       {query.allProjects.edges.map(({ node }) => (
-        <Card title={node.cifIdentifier} key={node.id}>
+        <Card title={node.uniqueProjectId} key={node.id}>
```

Comment:
...I never thought about this being a possibility! I'm not sure, I quite like knowing for sure if we're talking about Relay or database entities, although since we're describing database columns in the form data, that could become quite hairy to keep clean

Reply:
I didn't think of that as a possibility either. Yes, I think we should turn it off

Reply:
Or, maybe better, we don't mess with postgraphile's name inflections, and use a camel-to-snake case conversion in postgres when applying the form changes, with something like: `regexp_replace(name, '([A-Z])','_\1', 'g'))`

Reply:
wrapping that regex in a `lower()` would work